src/main.rs

```diff
@@ -1,24 +1,20 @@
+mod transforms;
+
 use std::{fs, path::Path};
 
-use bodymovin::Bodymovin as Lottie;
-use iconimation::Template;
+use bodymovin::{layers, properties, shapes, Bodymovin as Lottie};
+use iconimation::shapes_for_glyph;
 use kurbo::Point;
+use kurbo::{Affine, Rect};
 use skrifa::{
     raw::{FontRef, TableProvider},
     MetadataProvider,
 };
 
+use crate::transforms::scale_transform;
+
 fn main() {
     let args: Vec<String> = std::env::args().skip(1).collect();
-    let lottie_files: Vec<_> = args
-        .iter()
-        .map(Path::new)
-        .filter(|p| {
-            p.extension()
-                .map(|ext| ext.eq_ignore_ascii_case("json"))
-                .unwrap_or_default()
-        })
-        .collect();
```

Comment:
Instead of deleting the ability to use templates lets just make it use your coded template if no other template is specified

Reply:
I'll merge and do that after